Module/randomize.py

```diff
@@ -54,6 +54,35 @@ def populateLocations(self, excludeWorlds):
 
         self._validLocationListDonald = [location for location in self._allLocationListDonald if not set(location.LocationTypes).intersection(excludeWorlds)]
 
+        late_item_weight = 1
+        early_item_weight = 1
+        insane = False
+        if item_difficulty == "Super Easy":
```

Comment:
late_game_weight should probably be closer to .01 to be more symmetrical with insane.

Reply:
The exact numbers can be tweaked (maybe after I get some data on item distribution like you suggested for insane), but  these difficulties are inherently unbalanced since normal is the neutral setting. Two settings below normal, and three settings above normal.

Reply:
sure thats fine